src/SimpleCalculator.Console/Calculator.cs

```diff
@@ -25,28 +19,45 @@ public static void Main(string[] args)
                 Console.WriteLine("3. Multiplication");
                 Console.WriteLine("4. Division");
                 userOption = Convert.ToInt32(Console.ReadLine());
-
+                IUserInputValidation input = serviceProvider.GetService<IUserInputValidation>();
+                double num1;
+                double num2;
                 switch (userOption)
                 {
-
                     case 1:
                         var addService = serviceProvider.GetService<IAddition>();
-                        addService?.AddWorkFlow();
+                        Console.WriteLine("Please enter the first number");
```

Comment:
nitpick: there is a duplication of the first 3 lines. Could you please refactor it?